src/components/pages/landing/header/index.tsx

```diff
@@ -7,15 +7,33 @@ import HeroBgLight from './hero-bg-light.png'
 import HeroBgDark from './hero-bg-dark.png'
 import Image from 'next/image'
 import TechLogos from '../tech-logos'
+import title from 'title'
 
-const Header = () => {
+const Header: React.FC<{topic?: string; customer?: any}> = ({
+  topic,
+  customer,
+}) => {
   const {viewer} = useViewer()
   const {resolvedTheme} = useTheme()
   const [isMounted, setIsMounted] = React.useState(false)
   React.useEffect(() => {
     setIsMounted(true)
   }, [])
 
+  let Offer = () => (
+    <>
+      <CreateAccount location="homepage header" />
+      <p className="max-w-sm pt-10 text-xs text-center sm:text-sm opacity-60">
+        Enter your email to create an account and start learning from more than
+        3,000 free lessons on egghead.
+      </p>
+    </>
+  )
+
+  if (viewer || customer) {
```

Comment:
if we have a customer we don't need an email address so show them the "go pro" cta

this could be a lot more nuanced. Have they watched a video? Have they searched? What's the "current offer" that sends them down the value path?